### ISSUE TITLE
feature

### ISSUE THREAD

**Issue by wuqing778** [KrishnaDN/Emotion-Identification-from-Raw-Speech-Signals-using-TDNNs#4]
Hello, I have reproduced your code, but I don't quite understand one thing. May I ask that the feature you used in your code is the original waveform without any feature extraction, but the time domain, right?I don't think it's spectrogram or MFCC. May I ask if my understanding is correct? Thank you

**Comment by hemarathore** [comment]
Yes, the input to the network is a raw waveform. But may I ask if you have reproduced the code wat is the purpose of testing and training files, In the training file the testing is also done right?
 pLease explain :)
